demos/README.md

```diff
@@ -0,0 +1,35 @@
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah-logo_large.png)
+
+# Buildah Demos
+
+The purpose of these demonstrations is twofold:
+
+1. To help automate some of the tutorial material so that Buildah newcomers can walk through some of the concepts.
+2. For Buildah enthusiasts and practitioners to use for demos at educational presentations - college classes, Meetups etc.
+
+It is assumed that you have installed Buildah and Podman on you machine.
+
+    $ sudo dnf -y install podman buildah
+
+## Building from scratch demo 
+
+filename: `buildah-scratch-demo.sh`
```

Comment:
Can you make this a link to the file in GitHub somehow?

Reply:
I can make it explicitly link. It will break on any fork though, including my own.